src/lib/containers/entity/annotations/CustomDateTimeWidget.tsx

```diff
@@ -14,13 +14,9 @@ export const CustomDateTimeWidget: Widget = ({
   return (
     <CalendarWithIconFormGroup
       disabled={disabled}
-      value={value ? moment(value) : ''}
-      setterCallback={(newValue: string | Moment) => {
-        if (newValue == null || typeof newValue === 'string') {
-          onChange(newValue)
-        } else {
-          onChange(newValue.toISOString())
-        }
+      value={value ? dayjs(value) : ''}
+      setterCallback={(newValue: string | Dayjs | null) => {
+        onChange(newValue)
```

Comment:
Removed `onChange(newValue.toISOString()` as it was throwing errors when typing in dates with keyboard.

Reply:
What happens if it's a Dayjs object? I think `onChange` is expecting a `string` or `null`, even if the type accepts `Dayjs`

Reply:
If you mean when using the DateTimePicker it works. As soon as the user types into the input an error is thrown `Invalid time value`.

Reply:
We decided we'll open a follow-up ticket.